src/main/scala/dpla/ingestion3/MappingEntry.scala

```diff
@@ -57,10 +87,7 @@ object MappingEntry {
 
     // Load the harvested record dataframe
     val harvestedRecords: DataFrame = spark.read.avro(dataIn)
-
-    // Take the first harvested record in the dataFrame and get the provider value
```

Comment:
This is removed in favor of passing in the shortName via command line args so the provider-specific properties for Spark can be loaded before creating the SparkSession